index.js

```diff
@@ -0,0 +1,32 @@
+let n = 16;
+const Container = document.getElementById("container");
```

Comment:
Use camelCase for naming variables and functions.
So, `Container` will be `container`